### PR TITLE
Add authorization resources

### PR DESCRIPTION
**Pull request by irmatov**
* Role definitions
* Role assignments

### CODE REVIEW

File: resources/authorization_role_definitions.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:        "azure_authorization_role_definition_permissions",
+				Description: "Permission role definition permissions",
+				Resolver:    fetchAuthorizationRoleDefinitionPermissions,
+				Columns: []schema.Column{
+					{
+						Name:        "role_definition_id",
```
Comment (fdistorted) [inline resources/authorization_role_definitions.go:75]: role_definition_cq_id should be here

**Review by fdistorted — APPROVED** [review]
LGTM

File: resources/authorization_role_definitions.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+						Description: "Unique ID of azure_authorization_role_definitions table (FK)",
```
Comment (roneli) [inline resources/authorization_role_definitions.go:76]: Unique CloudQuery ID

**Review by roneli — APPROVED** [review]
LGTM, fix @fdistorted comment and we can merge

File: resources/authorization_role_definitions.go (hunk 1 above)
Comment (irmatov) [reply resources/authorization_role_definitions.go:75]: Good catch, thanks!

File: resources/authorization_role_definitions.go (hunk 2 above)
Comment (irmatov) [reply resources/authorization_role_definitions.go:76]: Updated

**Pull request merged**